plugins/kobolight.koplugin/main.lua

```diff
@@ -72,6 +72,12 @@ end
 function KoboLight:resetLayout()
     local new_screen_height = Screen:getHeight()
     self.gestureScale = new_screen_height * swipe_touch_zone_ratio.h * 0.8
+    local powerd = Device:getPowerDevice()
+    local scale = (powerd.fl_max - powerd.fl_min) / 20
+    for i = 1, #self.steps, 1
+    do
+        self.steps[i] = math.floor(self.steps[i] * scale)
```

Comment:
If you prefer to keep the existing behavior, I am happy to remove this change. I am just a little bit curious, how the levels come from. It looks too small for kobo, but good for kindle, which has only 24 levels.